### PR TITLE
Cancel booking

### LINKED ISSUE #17: Renter Dashboard - Cancel Booking
As a Renter, when I visit the Dashboard, I want to see my upcoming and pending bookings, so that I can click a button to cancel the booking if within the SITE POLICY cancellation timeframe.  I am redirected to my dashboard where I no longer see the booking.
 - Delete record from bookings table

### LINKED ISSUE #23: Renter Dashboard - Cancel Booking not Available within X hours
As a Renter, when I visit the Dashboard, I want to see my upcoming and pending bookings, where the "Cancel Booking" button is not available if the booking is within the next <Site Policy> number of hours.

### PR DESCRIPTION
**Pull request by jbranham1**
Added styling
Added the ability to cancel booking if it is not within 48 hours.

Close #17
Close #23

### CODE REVIEW

File: spec/features/host/dashboard/index_spec.rb [hunk 1]
```diff
@@ -141,5 +141,44 @@
...
+      #                     :time=>DateTime.now,
+      #                     :duration=>"2",
+      #                     :description=>"description"}
+      #       es = EngineService.create_booking(booking_params)
+      #       visit host_dashboard_index_path
+      #       within "#booking-#{es[:data][:id]}" do
+      #         expect(page).to_not have_button("Cancel Booking")
+      #       end
+      #   end
+      # end
```
Comment (abreaux26) [inline spec/features/host/dashboard/index_spec.rb]: @jbranham1 do you mind deleting this, or will we need it later?

Comment (jbranham1) [reply spec/features/host/dashboard/index_spec.rb]: Deleted!

**Review by abreaux26 — APPROVED** [review]
This is awesome

File: app/facades/booking_facade.rb [hunk 2]
```diff
@@ -31,4 +31,10 @@ def self.total_cost(duration, price)
   def self.yard_info(yard_id)
     EngineService.yard_details(yard_id)
   end
+
+  def self.get_time(booking)
+    d = booking[:attributes][:date].to_date
+    t = booking[:attributes][:time].to_time
+    dt = DateTime.new(d.year, d.month, d.day, t.hour, t.min)
+  end
```
Comment (amsmyth1) [inline app/facades/booking_facade.rb:39]: nice!

**Review by amsmyth1 — APPROVED** [review]
Looks good! Thanks for getting the styling updated

**Pull request merged**